NebulaHost/PacketProcessors/Logistics/ILSgStationPoolSyncRequestProcessor.cs

```diff
@@ -0,0 +1,155 @@
+﻿using NebulaModel.Attributes;
+using NebulaModel.Networking;
+using NebulaModel.Packets.Logistics;
+using NebulaModel.Packets.Processors;
+using NebulaModel.DataStructures;
+
+/*
+ * Whenever a client connects we sync the current state of all ILS and ships to them
+ * resulting in a quite large packet but its only sent one time upon client connect.
+ */
+namespace NebulaHost.PacketProcessors.Logistics
+{
+    [RegisterPacketProcessor]
+    public class ILSgStationPoolSyncRequestProcessor : IPacketProcessor<ILSRequestgStationPoolSync>
+    {
+        private PlayerManager playerManager;
+        private const int maxShipsPerILS = 10;
```

Comment:
Can this be unified with `const int ILSMaxShipCount = 10;` which is in the `NebulaClient.PacketProcessors.Logistics.ILSgStationPoolSyncProcessor`?

For example having this in both cases: `ILSShipManager.ILSMaxShipCount`